config/traits.yml

```diff
@@ -4329,6 +4334,7 @@ traits:
         andromonoecious: Taxa with hermaphrodite flowers and staminate flowers (producing
           just male gametes) on the same plant but no pistillate (female) flowers.
         dioecious: Taxa that produce male and female flowers on separate plants.
+        non-dioecious: Broad category to capture any taxa that are scored as not dioecious.
```

Comment:
Should we remove this again, since now we're not using it and we don't really want to encourage it as a term?

Reply:
Yes, I think so. I removed it.